src/Handler/Table/Drop/DropTableHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Drop;
+
+use Doctrine\DBAL\Exception;
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Table\DropTableCommand;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Teradata\ConnectionFactory;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Table\Teradata\TeradataTableQueryBuilder;
+
+final class DropTableHandler implements DriverCommandHandlerInterface
+{
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof DropTableCommand);
+
+        // validate
+        assert($command->getPath()->count() > 0, 'DropTableCommand.path is required');
```

Comment:
vynutil bych, že path = 1 nic jiného do teradaty nedojde a když jo je to chyba. Pak tu path nemusíš iterovat.

Reply:
ok, hotovo